app/helpers/pagelets_helper.rb

```diff
@@ -0,0 +1,35 @@
+module PageletsHelper
+  def pagelets_for(mountpoint)
+    Pagelets::Manager.sorted_pagelets_at("#{controller_name}/#{action_name}", mountpoint)
+  end
+
+  def render_pagelets_for(mountpoint, opts = {})
+    result = ""
+    pagelets_for(mountpoint).each do |pagelet|
+      result += render_pagelet(pagelet, opts)
+    end
+    result.html_safe
+  end
+
+  def render_tab_content_for(mountpoint, opts = {})
+    result = ""
+    pagelets_for(mountpoint).each do |pagelet|
+      result += "<div id='#{pagelet.snake_name}' class='tab-pane'>"
```

Comment:
I discovered one problem. The `snake_name` should be provided only as a default id with a possibility to override. Currently, javascript fails to switch to pagelet tabs that have name conflicting with an id of a different element in the page structure. For example it's impossible to show pagelet named "Content", because it conflicts with the main '#content' div.